### PR TITLE
[Server42] script for generate protobuf java files

### PR DESCRIPTION
**Pull request by nikitamarchenko**

### CODE REVIEW

File: src-cpp/server42/Dockerfile.protobuf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
```
Comment (surabujin) [inline src-cpp/server42/Dockerfile.protobuf:1]: can we have 20.04? Recently we have an issue with "missing" ubuntu repositories for outdated ubuntu release.

File: src-cpp/server42/Dockerfile.protobuf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+WORKDIR /root
+
+RUN echo 'deb http://archive.ubuntu.com/ubuntu/ bionic-proposed main' > /etc/apt/sources.list.d/bionic-proposed.list \
+  && apt-get update -y && apt-get install -y curl unzip
+
+RUN curl -L https://github.com/protocolbuffers/protobuf/releases/download/v3.8.0/protoc-3.8.0-linux-x86_64.zip --output protobuf.zip
```
Comment (surabujin) [inline src-cpp/server42/Dockerfile.protobuf:8]: Why not use `wget` (it should be available in the base image)?

File: src-java/server42/server42-stats-messaging/src/main/java/org/openkilda/server42/stats/messaging/flowrtt/Statistics.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: server42/statistics.proto
+// source: statistics.proto
```
Comment (surabujin) [inline src-java/server42/server42-stats-messaging/src/main/java/org/openkilda/server42/stats/messaging/flowrtt/Statistics.java:2]: Should it be removed from repo now?

**Review by surabujin — APPROVED** [review]

File: src-cpp/server42/Dockerfile.protobuf (hunk 1 above)
Comment (niksv) [reply src-cpp/server42/Dockerfile.protobuf:1]: +1

Comment (niksv) [reply src-cpp/server42/Dockerfile.protobuf:1]: +1

Comment (niksv) [reply src-cpp/server42/Dockerfile.protobuf:1]: agree

Comment (niksv) [reply src-cpp/server42/Dockerfile.protobuf:1]: agree

**Review by sirmax123 — APPROVED** [review]

File: src-cpp/server42/generate_java_protobuf.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+if [ ! -f "src-java" ]; then
```
Comment (sergii-iakovenko) [inline src-cpp/server42/generate_java_protobuf.sh:3]: I think it should be "-d" instead of "-f".

File: src-cpp/server42/Dockerfile.protobuf (hunk 2 above)
Comment (nikitamarchenko) [reply src-cpp/server42/Dockerfile.protobuf:8]: `/bin/sh: 1: wget: not found`

File: src-cpp/server42/Dockerfile.protobuf (hunk 1 above)
Comment (nikitamarchenko) [reply src-cpp/server42/Dockerfile.protobuf:1]: done

File: src-java/server42/server42-stats-messaging/src/main/java/org/openkilda/server42/stats/messaging/flowrtt/Statistics.java (hunk 3 above)
Comment (nikitamarchenko) [reply src-java/server42/server42-stats-messaging/src/main/java/org/openkilda/server42/stats/messaging/flowrtt/Statistics.java:2]: We need to discuss that. I will do it with a separate PR if somebody needs that.

File: src-cpp/server42/generate_java_protobuf.sh (hunk 4 above)
Comment (nikitamarchenko) [reply src-cpp/server42/generate_java_protobuf.sh:3]: good catch

**Pull request merged**